index.html

```diff
@@ -83,17 +77,17 @@ <h4>Update Chromebrew packages:</h4>
         <div class="col-12">
           <div class="bs-example bs-example-type">
             <h4>Install Chromebrew (along with Ruby and Git).</h4>
-            <h5>Supported on Chrome OS systems running on an ARM (ARMv7+), legacy 32-bit x86 (without GUI support) or x86-64 processor.</h5>
+            <h5>Supported on Chrome OS systems running on an ARM (ARMv7+) or x86-64 processor.</h5>
```

Comment:
@uberhacker I'm ok with this ...

Reply:
I think we should continue to provide binaries for the packages that can be built for i686 but it's fine to remove this from the sentence.

Reply:
But there is nobody still using `i686` as their daily driver in 2023... (even Windows dropped `i686` support nowadays, why are we still supporting it for nobody :) )

Reply:
I'm using it and keeping it alive will prevent all the old Chromebooks from becoming paper weights.  I just don't spend a lot of effort on packages that don't build.